packages/big-design/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -257,6 +257,15 @@ export const MultiSelect = typedMemo(
       stateReducer: handleStateReducer,
     });
 
+    let comboboxProps = getComboboxProps();
+
+    if (!label && props['aria-label']) {
```

Comment:
I think would be good to make `label` prop required and create a prop  called `hideLabel`, and if `hideLabel` is true we can internally assign 'aria-label' with the label value. Thoughts?

Reply:
I would like to avoid modifying the props if I can avoid it. 😅  The `label` prop being optional and devs having the option to use `aria-label` is already the behaviour available for other controls like `<Input>` and `<Select>`.

Reply:
>I see 👀 , I suggested this since I think would be good for us worrying about accessibility instead of the user, but up to us 👍🏼
I do like the idea of the library enforcing accessibility with required fields but I feel this is probably a pattern we would want consistently across all our components then right? 

> Would it be good to add a comment in the docs for MultiSelect mentioning that this component supports all its element attributes (or something you think might be helpful for the user to know)? 🍹 similar like the input component does: https://developer.bigcommerce.com/big-design/input

Yea agreed. Would be nice to document something like `Props of the native input component are also available`.
I might whip up a quick separate PR for this